File: Java/Akash-212/1732. Find the Highest Altitude.java
class Solution {
    public int largestAltitude(int[] arr) {
        List<Integer>lis=new ArrayList<>();
        lis.add(0);
         int j=0;
        for(int i=0;i<arr.length;i++){
            int addvalue=0;
            addvalue+= lis.get(j)+arr[i];
            j++;
            lis.add(addvalue);
        }
        System.out.println("the altitudes are :"+lis);
        int max=Collections.max(lis);
        return max;
    }
}

File: CPP/coolakt/1732. Find the Highest Altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
       int alt=0;
       int hgtalt=0;
       for(int i=0; i<gain.size(); i++){
           alt=alt+gain[i];
           hgtalt=max(alt,hgtalt);
       } 
       return hgtalt;
    }
};
